modules/local/samplesheet_check.nf

```diff
@@ -11,15 +11,15 @@ process SAMPLESHEET_CHECK {
 
     output:
     path '*.csv'       , emit: csv
-    path "versions.yml", emit: versions
+    path "software_versions.yml", emit: software_versions
 
     script: // This script is bundled with the pipeline, in nf-core/rnadeseq/bin/
     """
     check_samplesheet.py \\
```

Comment:
If you're using this to check the validity of the software_versions.file maybe you could rename the process `SAMPLESHEET_CHECK`  to `SOFTWARE_VERSIONS_CHECK`

Reply:
But in principle it would be nice if you could also check the validity of the input samplesheet